homeassistant/components/fronius/sensor.py

```diff
@@ -155,6 +155,27 @@ class FroniusSensorEntityDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.CURRENT,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    FroniusSensorEntityDescription(
+        key="current_ac_phase_1",
```

Comment:
In another PR for Fronius (that's not merged currently) I have added `translation_key` and `translation_placeholders` attributes for such keys - so we only have to do translation once for all 3 phases. This could be used here too, once merged. See https://github.com/home-assistant/core/pull/140120/files